packages/plugin-evm/src/tests/wallet.test.ts

```diff
@@ -23,9 +24,8 @@ describe('Wallet provider', () => {
 
   beforeAll(() => {
     pk = generatePrivateKey();
-
-    const chainNames = ['iotex', 'arbitrum'];
-    chainNames.forEach((chain) => (customChains[chain] = WalletProvider.genChainFromName(chain)));
+    // Add the custom chain to the customChains object
+    customChains['myCustomChain'] = customChain;
```

Comment:
As this string `'myCustomChain'` is used more than once, let's extract it to variable